src/constants.js

```diff
@@ -0,0 +1,7 @@
+export const SIDEBAR_EXPAND = "SIDEBAR_EXPAND";
```

Comment:
Hi @icaromh, Could you please put the copyright holder and license inside this file?